openapi-spec/swagger.yaml

```diff
@@ -476,6 +476,32 @@ paths:
           description: The resource does not exist.
         '500':
           $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{projectId}/volumes/{volumeId}/action':
```

Comment:
Can you remove the definition in open-api for now, because this feature is at alpha stage. We should think it over before push it to the standard